### PR TITLE
removing sqlalchemy definitions and loading them via library hapi-schema

### PR DESCRIPTION
**Pull request by SimonbJohnson**
The only problem I see is that hapi-schema package did not get added to the requirements.txt when I reran pip-compile.  I could add it manually, but assume this will be deleted when the command is re-run. 

Use
`pip install hapi-schema`
to install it

### CODE REVIEW

**Review by mcarans — CHANGES_REQUESTED** [review]
I think you need to add the hapi-schema library here:
https://github.com/OCHA-DAP/hapi-pipelines/blob/649a909ccae57f89591d5d62b09c3dabc26d13ca/pyproject.toml#L36-L41

Then pip-compile run automatically within the pre-commit hook will add it to the requirements.txt file.

File: src/hapi/pipelines/app/__main__.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def main(
     else:
         scrapers_to_run = None
     project_config_yaml = script_dir_plus_file(
-        "project_configuration.yaml", main
+        "project_configuration .yaml", main
```
Comment (turnerm) [inline src/hapi/pipelines/app/__main__.py:155]: typo?

**Comment by SimonbJohnson** [comment]
That change has now been made

**Review by turnerm — APPROVED** [review]
Looks good to me pending my small comment and the changes that Mike requested

File: src/hapi/pipelines/app/__main__.py (hunk 1 above)
Comment (SimonbJohnson) [reply src/hapi/pipelines/app/__main__.py]: Yes, good spot, corrected!

**Pull request merged**